api/samples/CMakeLists.txt

```diff
@@ -206,10 +206,10 @@ configure_DynamoRIO_global(OFF ON)
 add_sample_client(bbbuf       "bbbuf.c"         "drmgr;drreg;drx")
 add_sample_client(bbsize      "bbsize.c"        "drmgr")
 add_sample_client(empty       "empty.c"         "")
+add_sample_client(bbcount     "bbcount.c"       "drmgr;drreg")
```

Comment:
These used to all be in alphabetical order.   The arm porting and grouping of unported samples has split up the order.  Let's get it back to alphabetical for the ported ones at least by putting this where it was before the arm port, up between bbbuf and bbsize.